src/gennylib/src/PoolManager.cpp

```diff
@@ -38,6 +39,12 @@ auto createPool(const std::string& name,
         poolFactory.setOptions(genny::v1::PoolFactory::kAccessOption, *accessOpts);
     }
 
+    auto encryptOpts = context["Clients"][name]["EncryptionOptions"].maybe<EncryptionOptions>();
+    if (encryptOpts) {
+        poolFactory.setEncryptionContext(
+            encryptionManager.createEncryptionContext(mongoUri, *encryptOpts));
```

Comment:
Am I correct in understanding that the mongoUri used here doesn't have all the options set that PoolFactory would have set? Do we want it to, in case some of the options spliced in there are needed for the connection, and we could re-path some of this so that the URI doesn't get used until a call inside PoolFactory?

Reply:
I changed it to use the output of `poolFactory.makeUri()` instead.